independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/tooling/dockerfiles/codestart.yml

```diff
@@ -5,6 +5,6 @@ language:
     data:
       dockerfile:
         native:
```

Comment:
Can we add a comment (it's yaml, so we can) explaining why here we want to stay on UBI8 and not switch to UBI9?